### ISSUE TITLE
Replace INSTALL with apt get grpc

### ISSUE THREAD

**Issue by oaktowner** [grpc/grpc-common#80]
In the Node.js sample, replace the INSTALL (and build) of the grpc binaries with an apt get grpc as soon as the package is available.

**Comment by ryanseys** [comment]
Are there going to be Mac binaries available as well? I'm running into an issue while attempting to `npm install` that might be fixed with a binary or installation of the appropriate libraries.

``` 
$ npm install
> grpc@0.5.0 install /Users/ryanseys/node_modules/grpc
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/grpc/ext/byte_buffer.o
../ext/byte_buffer.cc:39:10: fatal error: 'grpc/grpc.h' file not found
#include "grpc/grpc.h"
```

**Comment by tbetbetbe** [comment]
The installation on Mac should be resolved by the new instructions added in #225
